pkg/sink/sink_test.go

```diff
@@ -498,12 +498,10 @@ func TestResource_validateEvent(t *testing.T) {
 		EventListenerNamespace: "foo",
 	}
 
-	triggerValidate := &triggersv1.TriggerValidate{
-		TaskRef: pipelinev1.TaskRef{
-			Name: "bar",
-		},
-		ServiceAccountName: "foo",
-	}
+	triggerValidate := &triggersv1.TriggerValidate{}
```

Comment:
The top level validate builder `EventListenerTriggerValidate()` is not used here